src/guides/neutral/skinchanger.mdx

```diff
@@ -0,0 +1,41 @@
+---
+name: Skinchanger
```

Comment:
#211

Reply:
Updated to hopefully reach a good version that covers all of the ground we both wanted to cover, using pieces of both versions. Hans has expressed satisfaction with this.

Reply:
@TheHans255 you'll need to update/recreate the rest of your PR once this is merged in.